flexmeasures/ui/templates/crud/users.html

```diff
@@ -48,6 +49,9 @@ <h2>All {% if not include_inactive %}active {% endif %}users</h2>
                                       {{ role.name }}{{ "," if not loop.last }}
                                     {% endfor %}
                                 </td>
+                                <td>
+                                    {{ user.account.name }}
```

Comment:
I believe this page would crash given a user without an account. Even though we don't expect to be in that situation, should we build in a fail-safe? For example, by checking here whether user.account is not None, or giving the user class a property `account_name` (and referring to that here) which does the relevant check in one central place (returning something like "no account known" in case the user is not known to be connected to an account). I prefer the latter option.

Reply:
`User.accouunt_id` is `nullable=False` so it really can't imagine why that should happen.